api/src/controllers/publications.js

```diff
@@ -221,10 +222,14 @@ async function scrapeGoogleScholar(url) {
   const context = await browser.newContext();
   const page = await context.newPage();
   await page.goto(playwrightConfig.gScholarHome + url);
-
-  const title = await page.$$eval('a.gsc_oci_title_link', (titles) =>
+  let title = await page.$$eval('a.gsc_oci_title_link', (titles) =>
     titles.map((title) => title.innerText)
-  );
+  ); // we assume the publication title is a link
+
+  if (title[0] === undefined) { // if its undefined, then it wasn't a link
+    title = await page.$$eval('div[id=gsc_oci_title]',(titles) =>
```

Comment:
See example of publication with no link here: https://scholar.google.com/citations?view_op=view_citation&hl=en&user=bbEQGY8AAAAJ&sortby=pubdate&citation_for_view=bbEQGY8AAAAJ:GbL5_m8SeH4C